File: Java/백준/Bronze/17350. 2루수 이름이 뭐야/2루수 이름이 뭐야.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        for (int i=0;i<a;i++){
            String s = in.next();
            if (s.equals("anj")){
                System.out.println("뭐야;");
                break;
            }
            if (i==a-1){
                System.out.println("뭐야?");
            }
        }
    }
}

File: C#/백준/Bronze/17350. 2루수 이름이 뭐야/2루수 이름이 뭐야.cs
using System;

namespace q{
    class w{
        static void Main(String[] args){
            int a =  int.Parse(Console.ReadLine());
            for (int i=0;i<a;i++){
                string s = Console.ReadLine();
                if (s.Equals("anj")){
                    Console.WriteLine("뭐야;");
                    break;
                }
                if (i==a-1){
                    Console.WriteLine("뭐야?");
                }
            }
        }
    }
}